thirdparty/libktx/Apache-2.0.txt

```diff
@@ -0,0 +1,208 @@
+Apache License
```

Comment:
@akien-mga is apache 2 compatible with MIT? (for godot)

Reply:
Yes, we already use Apache 2-licensed dependencies.